### DISCUSSION TITLE
4/14 v-if, v-show, v-for

### DISCUSSION THREAD

**Discussion by May0505** [Tech-Book-Community/Vue3#7] [General]
簡報：https://hackmd.io/Mu2bzxK9Q2SLu7CFU6piAw?view

**Comment by janlin002** [comment]
`v-if` vs `v-show`

https://vuejs.org/guide/essentials/conditional.html#v-if-vs-v-show

> v-if has higher toggle costs while v-show has higher initial render costs. So prefer v-show if you need to toggle something very often, and prefer v-if if the condition is unlikely to change at runtime.

**Reply by janlin002** [reply]
摺疊選單打開裡面  用 v-if 資料就會被清掉 (??

**Comment by janlin002** [comment]
`v-for` 為什麼需要綁定 `key` ?

如果沒有綁定，會有什麼問題？
